pkg/auth/globalrole.go

```diff
@@ -65,9 +65,60 @@ func (g *GlobalRoleResolver) ClusterRulesFromRole(gr *v3.GlobalRole) ([]rbacv1.P
 		}
 		rules = append(rules, templateRules...)
 	}
+
 	return rules, nil
 }
 
+func (g *GlobalRoleResolver) FleetWorkspacePermissionsResourceRulesFromRole(gr *v3.GlobalRole) []rbacv1.PolicyRule {
```

Comment:
This exported function should have a comment describing what it's for.

Reply:
you're right! done in https://github.com/rancher/webhook/pull/348/commits/57b1a9fe5620b0e9b32d2368caaad5834c9ec84d